### PR TITLE
Update .travis.yml file to use tox as script for running tests

### PR DESCRIPTION
**Pull request by dan98765**
Goal: use tox script in travis to run tests.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,64 +1,27 @@
...
+       python: 3.5
+    -  env: TOXENV=py36
+       python: 3.6
+    -  env: TOXENV=pypy
+       python: pypy-5.7.1
+    -  env: TOXENV=pre-commit
+       python: 3.6
+install:
+  - pip install coveralls tox
+  - if [[ $TRAVIS_PYTHON_VERSION == '3.6' ]]; then pip install mypy; fi
```
[the remark this answers is not in the document]
Comment (dan98765) [reply .travis.yml]: Good catch, thanks. Added!

**Comment by dan98765** [comment]
@syrusakbary FYI this is the graphene repo's version of https://github.com/graphql-python/graphql-core/pull/188

**Comment by dan98765** [comment]
@syrusakbary could you please give this a review?

File: .travis.yml [hunk 2]
```diff
@@ -1,64 +1,27 @@
...
-after_success:
-- |
-  if [ "$TEST_TYPE" = build ]; then
-      coveralls
-  fi
-env:
-  matrix:
-  - TEST_TYPE=build
-  global:
-    secure: SQC0eCWCWw8bZxbLE8vQn+UjJOp3Z1m779s9SMK3lCLwJxro/VCLBZ7hj4xsrq1MtcFO2U2Kqf068symw4Hr/0amYI3HFTCFiwXAC3PAKXeURca03eNO2heku+FtnQcOjBanExTsIBQRLDXMOaUkf3MIztpLJ4LHqMfUupKmw9YSB0v40jDbSN8khBnndFykmOnVVHznFp8USoN5F0CiPpnfEvHnJkaX76lNf7Kc9XNShBTTtJsnsHMhuYQeInt0vg9HSjoIYC38Tv2hmMj1myNdzyrHF+LgRjI6ceGi50ApAnGepXC/DNRhXROfECKez+LON/ZSqBGdJhUILqC8A4WmWmIjNcwitVFp3JGBqO7LULS0BI96EtSLe8rD1rkkdTbjivajkbykM1Q0Tnmg1adzGwLxRUbTq9tJQlTTkHBCuXIkpKb1mAtb/TY7A6BqfnPi2xTc/++qEawUG7ePhscdTj0IBrUfZsUNUYZqD8E8XbSWKIuS3SHE+cZ+s/kdAsm4q+FFAlpZKOYGxIkwvgyfu4/Plfol4b7X6iAP9J3r1Kv0DgBVFst5CXEwzZs19/g0CgokQbCXf1N+xeNnUELl6/fImaR3RKP22EaABoil4z8vzl4EqxqVoH1nfhE+WlpryXsuSaF/1R+WklR7aQ1FwoCk8V8HxM2zrj4tI8k=
```
[the remark this answers is not in the document]
Comment (syrusakbary) [reply .travis.yml]: We should keep this value. Is used to automatically upload the package to PyPI

**Comment by syrusakbary** [comment]
Will approve and merge once the global `secure` var is re-added :)

**Pull request merged**